arch/Kconfig

```diff
@@ -40,6 +40,7 @@ config X86
 	select HAS_DTS
 	select ARCH_HAS_CUSTOM_SWAP_TO_MAIN if !X86_64
 	select CPU_HAS_MMU
+	select ARCH_HAS_GDBSTUB
```

Comment:
Needs `if !X86_64` if it is just for 32-bit.

Reply:
done, thanks !